examples/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs.ipynb

```diff
@@ -60,25 +60,26 @@
      "name": "stdout",
```

Comment:
<blockquote>Amazon Braket Hybrid Jobs provides fully managed execution of hybrid quantum-classical algorithms, combining AWS classical compute resources based on Amazon EC2 (the "job instance") with Amazon Braket quantum processing units (QPUs) or quantum circuit simulators. </blockquote><p>Probably no longer necessary; instead, continue from the previous notebook, something like "This time, we will be passing three arguments to the <code>@hybrid_job</code> decorator. First, ... "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>okay I cut it now </p>